python_modules/dagster/dagster/_core/run_coordinator/queued_run_coordinator.py

```diff
@@ -133,6 +158,24 @@ def config_type(cls):
                 is_required=False,
                 description="If dequeue_use_threads is true, limit the number of concurrent worker threads.",
             ),
+            "max_user_code_failure_retries": Field(
+                config=IntSource,
+                is_required=False,
+                default_value=0,
```

Comment:
we expect to bump this up after we get some usage on this right?

Reply:
We could - the thing about it is that the number of run launchers that go through user code in OSS is actually quite low by design (today it's only the default run launcher)